frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -122,26 +122,16 @@ export class HtmlDocument extends React.Component {
     }
   }
 
-  handleContentCreated = data => {
-    const { state } = this
-    if (data.fields.content.parent_id !== state.content.content_id || data.fields.content.content_type !== 'comment') return
-
-    const sortedNewTimeline = sortTimelineByDate(
-      [
-        ...state.timeline,
-        {
-          ...data.fields.content,
-          created: displayDistanceDate(data.fields.content.created, state.loggedUser.lang),
-          created_raw: data.fields.content.created,
-          timelineType: 'comment',
-          hasBeenRead: data.fields.client_token === this.sessionClientToken
-        }
-      ]
-    )
+  handleContentCommentCreated = (tlm) => {
+    const { props, state } = this
+    // Not a comment for our content
+    if (tlm.fields.content.parent_id !== state.content.content_id) return
```

Comment:
`data.fields.content.content_type !== 'comment'` it's not necessary?

Reply:
No since the TLM handler has got the comment subtype.